workspace/workflows/jobs.py

```diff
@@ -141,6 +137,14 @@ def get_latest_conclusions(self) -> dict:
         }
         return conclusions
 
+    @staticmethod
+    def get_conclusion_for_run(run) -> str:
+        aliases = {"in_progress": "running"}
+        if run["conclusion"] is None:
+            status = str(str(run["status"]))
+            return aliases.get(status, status)
+        return str(run["conclusion"])
```

Comment:
Also here - it's not None, so can it be anything other than a string?
(Sorry for not commenting before, I know this method has just moved in this commit)

Reply:
Have removed it as you are correct - I think it was lingering from before the `if is None` statement was added.